NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 19.3.1 2020-07-31
+
+* 19.3.0 upgrade error "cannot change return type of existing function" (MODINVSTOR-547)
```

Comment:
```suggestion
* Removes `pmh_view_function` function prior to replacement during upgrade (MODINVSTOR-547)
```

Reply:
Given that this branch already includes the release itself, this feedback likely comes too late for this work :-/

Reply:
Good point Marc, I fixed this.

Reply:
You've fixed it after the release was performed, meaning that the tag refers to the old news or you have redone the maven release and replaced this branch (and the tag, if pushed)?

Reply:
The GitHub release page has been updated: https://github.com/folio-org/mod-inventory-storage/releases/tag/v19.3.1
The branch b19.3 contains the fix: https://github.com/folio-org/mod-inventory-storage/blob/b19.3/NEWS.md
I usually look at the NEWS.md file at the head of a branch and don't go back to some old commit.